llpc/context/llpcPipelineContext.cpp

```diff
@@ -130,6 +130,10 @@ static cl::opt<int> DontUnrollHintThreshold("dontunroll-hint-threshold",
                                             cl::desc("loop unroll threshold to use for loops with DontUnroll hint"),
                                             cl::init(0));
 
+// -lds-spill-limit-dwords: maximum amount of LDS space to be used for spilling
+static cl::opt<int> LdsSpillLimitDwords("lds-spill-limit-dwords",
+                                        cl::desc("Maximum amount of LDS space to be used for spilling"), cl::init(0));
```

Comment:
Can you explain how 0 or any negative number is interpreted here? For example, does 0 mean no spilling or no limit to spilling? (In the option description.)

Reply:
Clarification added.